tests/test_generated_examples.py

```diff
@@ -1601,3 +1601,27 @@ def test_checkout_session_create2(self, request_mock):
             ],
         )
         request_mock.assert_requested("post", "/v1/checkout/sessions")
+
+    def test_paymentintent_create2(self, request_mock):
```

Comment:
what's that?

Reply:
A codegenned test from a previous release that didn't affect the stripe-python implementation.